st2sdk/metagen/cmdmixin.py

```diff
@@ -0,0 +1,120 @@
+import argparse
+import six
+
+from cmd2 import options, make_option
+from st2sdk.metagen import metagen
+
+COMMAND_HELP = {
+    'metagen': [
+        'metagen [pack name] [module name]',
+        'Create initial directory structure for the provided pack. Inspect module and autogenerate'
+        'action metadata.'
+    ]
+}
+
+
+class Data(object):
```

Comment:
Its temp. Playing around with some options without actually changing metagen. I put it in place as metagen expects an object.